tools/vuxml/testdata/VUXML/CVE-2025-29087.json

```diff
@@ -0,0 +1,42 @@
+{
+    "schema_version": "1.7.0",
+    "id": "409206f6-25e6-11f0-9360-b42e991fc52e",
+    "modified": "2025-04-30T00:00:00Z",
+    "published": "2025-04-30T00:00:00Z",
+    "summary": "sqlite -- integer overflow",
+    "details": "\n\t\n\tcve@mitre.org reports:\n\t\n\t\n\tIn SQLite 3.44.0 through 3.49.0 before 3.49.1, the\n\tconcat_ws() SQL function can cause memory to be written\n\tbeyond the end of a malloc-allocated buffer. If the\n\tseparator argument is attacker-controlled and has a large\n\tstring (e.g., 2MB or more), an integer overflow occurs in\n\tcalculating the size of the result buffer, and thus malloc\n\tmay not allocate enough memory.\n\t\n\t\n\t\n    \n    ",
+    "references": [
+        {
+            "type": "ADVISORY",
+            "url": "https://api.osv.dev/v1/vulns/CVE-2025-29087"
+        },
+        {
+            "type": "ADVISORY",
+            "url": "https://nvd.nist.gov/vuln/detail/CVE-2025-29087"
+        }
+    ],
+    "affected": [
+        {
+            "package": {
+                "ecosystem": "FreeBSD:ports",
```

Comment:
In a separate PR, could you please add a ecosystem definition to https://ossf.github.io/osv-schema/#affectedpackage-field ?